Modelica/Electrical/PowerConverters.mo

```diff
@@ -6948,7 +6948,7 @@ General information about DC/DC converters can be found at the
     model EnableLogic
       "Partial model providing enable parameter and optional enable input"
       parameter Boolean useConstantEnable=true
-        "true = disabled boolean input, use constantEnable";
+        "Disable boolean input and use constantEnable, if true";
```

Comment:
Can write `=true, if constantEnable is utilized of firing signals, otherwise utilize the enable port`, i.e. start Boolean paramters with equality operator.